docs/modules/ROOT/pages/getting-started/filters-and-projections.adoc

```diff
@@ -1,130 +1,137 @@
+====
 
+. Finally, your `MATCH` clause should now look like this:
++
 ```javascript
 const clause = new Cypher.Match(movieNode)
     .where(Cypher.eq(movieNode.property("title"), new Cypher.Param("The Matrix")))
     .return(movieNode);
 ```
 
-=== Cypher and params
-
-By running the script again (`node main.js`), you should get the following Cypher:
-
+. Run the script again (`node main.js`). 
+You should now get the following Cypher:
++
 ```cypher
 MATCH (this0:Movie)
 WHERE this0.title = $param0
 RETURN this0
 ```
-
++
 That means `Movie` is now filtered by its title to a param, but the param itself is missing from the output.
 To fix that, you need to modify the build process at the end of the script:
-
++
 ```javascript
 const { cypher, params } = match.build();
 console.log(cypher);
 console.log(params);
 ```
-
-After executing it, you'll get both, the Cypher query and its parameters:
-
++
+Executing this script prompts the following Cypher query and its parameters:
++
 ```cypher
 MATCH (this0:Movie)
 WHERE this0.title = $param0
 RETURN this0
 ```
-
++
 ```javascript
 { param0: 'The Matrix' }
 ```
 
 === `AND` filtering
 
-To add the second filter, which needs to be concatenated with `AND`, use the method `.and` after the `where` with a new `Cypher.eq` predicate:
+Here is how to add an `AND` filter to your query:
 
+. Use the method `.and` after `.where`, with a new `Cypher.eq` predicate:
++
 ```javascript
     .where(Cypher.eq(movieNode.property("title"), new Cypher.Param("The Matrix")))
     .and(Cypher.lt(movieNode.property("released"), new Cypher.Param(2000)))
 ```
++
+This takes the same parameters as `.where` and ensures the proper concatenation of the filters with `AND`.
++
+In this case, the function `Cypher.lt` will add the operator `<` (**l**ower **t**han) and, like before, the comparison happens between a property of `movieNode` and a param.
 
-This takes the same parameters as `.where`, and makes sure to properly concatenate the filters with `AND`. In this case, the function `Cypher.lt` will add the operator `<` (**l**ower **t**han). Same as before, the comparison happens between a property of `movieNode` and a param.
-
-Running the script again:
-
+. Run the script again. You should get these results:
++
 ```cypher
 MATCH (this0:Movie)
 WHERE (this0.title = $param0 AND this0.released < $param1)
 RETURN this0
 ```
-
++
 ```javascript
 { param0: 'The Matrix', param1: 1999 }
 ```
-
++
 [NOTE]
 ====
-Adding the `AND` operation automatically adds parenthesis to ensure operation precedence, this will be important when dealing with complex and nested filters 
+The `AND` operation automatically adds parentheses to ensure operation precedence. 
+This is an important feature when dealing with complex and nested filters.
+See more information about xref:getting-started/relationships-and-advanced-filtering.adoc[Advanced filtering]. 
```

Comment:
I'm not sure about prompting the user to jump ahead in the tutorial. Maybe we could skip this link until we have a separate explanation or hot-to on this?

Reply:
But maybe what they want to learn is what we're considering more advanced. Do you think if they don't go through this page first they won't understand the other one on advanced filtering?

Reply:
Well, is mostly that the examples are based on this one, so it may be a bit confusing